otter/tap/api.py

```diff
@@ -227,6 +228,17 @@ def makeService(config):
         admin_service = service(str(admin_port), admin_site)
         admin_service.setServiceParent(s)
 
+    # setup cloud feed
+    cf_conf = config.get('cloudfeeds', None)
+    if cf_conf is not None:
+        service_configs[ServiceType.CLOUD_FEEDS] = {
```

Comment:
Because I want cloudfeeds to be optional. Especially in dev environment. It loses optional(ness) if it has to come from `get_service_configs` as that will always assume cloudfeed's config to be there. In fact, I feel `service_config` should be created in `makeService` depending on the values in config file instead of expecting all configs in `get_service_config`.